docs/help-topics/stonesoup-learnmore/stonesoup-configure-a-component/stonesoup-configure-components.yml

```
@@ -0,0 +1,27 @@
+# Name is an internal name. Title shows up in the UI as a side panel title.
+# Tags to be kept empty for now. Tags will specify where in the app descriptions will be available.
+# Links to be external only. We don't know yet whether referencing to other side panels will be supported but referencing to in-depth docs is expected to be supported.
+# Note that this topic is for LEARN MORE.
+
+- name: stonesoup-add-a-component
```

The name should be unique.
```suggestion
- name: stonesoup-configure-components
```